samples/snippets/src/main/java/aiplatform/UndeployModelSample.java

```diff
@@ -0,0 +1,79 @@
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_undeploy_model_sample]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.aiplatform.v1beta1.EndpointName;
+import com.google.cloud.aiplatform.v1beta1.EndpointServiceClient;
+import com.google.cloud.aiplatform.v1beta1.EndpointServiceSettings;
+import com.google.cloud.aiplatform.v1beta1.ModelName;
+import com.google.cloud.aiplatform.v1beta1.UndeployModelOperationMetadata;
+import com.google.cloud.aiplatform.v1beta1.UndeployModelResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class UndeployModelSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String endpointId = "YOUR_ENDPOINT_ID";
+    String modelId = "YOUR_MODEL_ID";
+    undeployModelSample(project, endpointId, modelId);
+  }
+
+  static void undeployModelSample(String project, String endpointId, String modelId)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    EndpointServiceSettings endpointServiceSettings =
+        EndpointServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (EndpointServiceClient endpointServiceClient =
+        EndpointServiceClient.create(endpointServiceSettings)) {
+      String location = "us-central1";
+      EndpointName endpointName = EndpointName.of(project, location, endpointId);
+      ModelName modelName = ModelName.of(project, location, modelId);
+
+      // key '0' assigns traffic for the newly deployed model
+      // Traffic percentage values must add up to 100
+      // Leave dictionary empty if endpoint should not accept any traffic
+      Map<String, Integer> trafficSplit = new HashMap<>();
+      trafficSplit.put("0", 100);
+
+      OperationFuture<UndeployModelResponse, UndeployModelOperationMetadata> operation =
+          endpointServiceClient.undeployModelAsync(
+              endpointName.toString(), modelName.toString(), trafficSplit);
+      System.out.format("Operation name: %s\n", operation.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      UndeployModelResponse undeployModelResponse = operation.get(180, TimeUnit.SECONDS);
```

Comment:
i see the other timeouts are set to be 300 but it is 180 here... any reason it is different? Maybe it is a good idea to link to any doc on setting timeout in samples that are doing so.

Reply:
Ok, undeploy LRO usually finishes faster other LROs.
```
Import Data: 5-30mins
Create Model: 8-24 hours
Deploy Model: 5-30mins
Undeploy model: 1-3min IIRC
```

Reply:
Add doc link to all the timeout code or specifically only for undeploymodel?

Reply:
I think it would be best if you just link to all of them! this is a piece of useful information.

Reply:
Currently we do not have an official docs yet, I will create a github issue to keep track of.

Reply:
https://github.com/googleapis/java-aiplatform/issues/16

Reply:
okay thanks - LGTM. Please feel free to merge whenever ready.